docs_user/modules/con_preparing-the-shared-file-systems-service-configuration.adoc

```diff
@@ -0,0 +1,120 @@
+[id="preparing-the-shared-file-systems-service-configuration_{context}"]
+
+= Recommendations for preparing the {rhos_component_storage_file} configuration
+
+To deploy {rhos_component_storage_file_first_ref} on the control plane, you must copy the original configuration file from the {rhos_prev_long} {rhos_prev_ver} deployment. You must review the content in the file to make sure you are adopting the correct configuration for {rhos_long} {rhos_curr_ver}. Not all of the content needs to be brought into the new cloud environment.
+
+Review the following recommendations for preparing your {rhos_component_storage_file} configuration file for adoption:
+
```

Comment:
@fmount @gouthampacha This is the new "preparing the configuration" section. Did I capture the purpose of this information accurately? That is, are these recommendations for preparing your manila service config file for adoption?